src/state/farms/fetchFarms.ts

```diff
@@ -8,88 +8,105 @@ import { FarmConfig } from 'config/constants/types'
 
-      // Amount of token in the LP that are staked in the MC (i.e amount of token * lp ratio)
-      const tokenAmountMc = tokenAmountTotal.times(lpTokenRatio)
-      const quoteTokenAmountMc = quoteTokenAmountTotal.times(lpTokenRatio)
+        // Amount of token in the LP that are staked in the MC (i.e amount of token * lp ratio)
+        const tokenAmountMc = tokenAmountTotal.times(lpTokenRatio)
+        const quoteTokenAmountMc = quoteTokenAmountTotal.times(lpTokenRatio)
 
-      // Total staked in LP, in quote token value
-      const lpTotalInQuoteToken = quoteTokenAmountMc.times(new BigNumber(2))
+        // Total staked in LP, in quote token value
+        const lpTotalInQuoteToken = quoteTokenAmountMc.times(new BigNumber(2))
 
-      const [info, totalAllocPoint] = await multicall(masterchefABI, [
-        {
-          address: getMasterChefAddress(),
-          name: 'poolInfo',
-          params: [farmConfig.pid],
-        },
-        {
-          address: getMasterChefAddress(),
-          name: 'totalAllocPoint',
-        },
-      ])
+        const [info, totalAllocPoint] = await multicall(masterchefABI, [
+          {
+            address: getMasterChefAddress(),
+            name: 'poolInfo',
+            params: [farmConfig.pid],
+          },
+          {
+            address: getMasterChefAddress(),
+            name: 'totalAllocPoint',
+          },
+        ])
 
-      const allocPoint = new BigNumber(info.allocPoint._hex)
-      const poolWeight = allocPoint.div(new BigNumber(totalAllocPoint))
+        const allocPoint = new BigNumber(info.allocPoint._hex)
+        const poolWeight = allocPoint.div(new BigNumber(totalAllocPoint))
 
-      return {
-        ...farmConfig,
-        tokenAmountMc: tokenAmountMc.toJSON(),
-        quoteTokenAmountMc: quoteTokenAmountMc.toJSON(),
-        tokenAmountTotal: tokenAmountTotal.toJSON(),
-        quoteTokenAmountTotal: quoteTokenAmountTotal.toJSON(),
-        lpTotalSupply: new BigNumber(lpTotalSupply).toJSON(),
-        lpTotalInQuoteToken: lpTotalInQuoteToken.toJSON(),
-        tokenPriceVsQuote: quoteTokenAmountTotal.div(tokenAmountTotal).toJSON(),
-        poolWeight: poolWeight.toJSON(),
-        multiplier: `${allocPoint.div(100).toString()}X`,
+        return {
+          ...farmConfig,
+          tokenAmountMc: tokenAmountMc.toJSON(),
+          quoteTokenAmountMc: quoteTokenAmountMc.toJSON(),
+          tokenAmountTotal: tokenAmountTotal.toJSON(),
+          quoteTokenAmountTotal: quoteTokenAmountTotal.toJSON(),
+          lpTotalSupply: new BigNumber(lpTotalSupply).toJSON(),
+          lpTotalInQuoteToken: lpTotalInQuoteToken.toJSON(),
+          tokenPriceVsQuote: quoteTokenAmountTotal.div(tokenAmountTotal).toJSON(),
+          poolWeight: poolWeight.toJSON(),
+          multiplier: `${allocPoint.div(100).toString()}X`,
+        }
       }
+
+      const timeout = new Promise((resolve) => {
```

Comment:
This seems reasonable Memoyil, might be worth adding a comment to explain why we're doing it and could you fix your prettier linting?

Reply:
Logic will be changed a bit after this https://github.com/pancakeswap/pancake-frontend/pull/1332 merged, so I will wait until that to visit your comment.

What is wrong with my linting? I am using yarn prettier directly

Reply:
Cool cool. Some unintended prettier changes in this PR I think? Maybe not... the diff is a bit difficult to read. https://github.com/pancakeswap/pancake-frontend/pull/1318/files#diff-c124a2935612b3ec404c2824c52fc4f4ff5ff25ffe1a92bbc2f7e2b697d30c99L49-R57

Reply:
That happened because I extracted that logic into variable. All fetch logic indented.